api/v1alpha1/openapi.yaml

```diff
@@ -2529,6 +2631,22 @@ components:
               type: array
               items:
                 type: string
+        resources:
```

Comment:
Just to confirm, we'd keep the `device.status.resources` with the summary status for each device resource.
And additionally, this new `device.spec.resources` would have the alarm details.

LGTM.